andela/ah-fortem-frontend
# 166980311 Add pagination support for articles

#### Title
This PR adds functionality for users to navigate through articles which are displayed in tens( meaning 10 articles per page.)

#### Description
Displaying all the articles at once to a user can lead to very bad user experience. Breaking articles into different pages allows users to view a given number of articles at a time.

#### How to manually test this:

1. Clone this repo and checkout the branch `story/166980311-pagination-support`
2. Run the command `npm run dev`
3. If the articles exceed 10, there should be a pagination bar at the bottom of the page that allows you to move to the next and previous pages

#### Type of change








#### How Has This Been Tested?

#### Checklist:
- [x] Add page numbers to the home page
- [x] Allow users to navigate through the pages

#### PT stories
[#166980311](https://www.pivotaltracker.com/story/show/166980311)
